<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
border: 1px solid black;
    border-radius: 15px;
    background-color: rgba(255, 255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">QFrame{
border-radius:15px;
background-color: black
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>391</width>
         <height>591</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFEC9E;
color:black;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="accountEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>230</y>
          <width>251</width>
          <height>35</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
        border: 1px solid #a1a1a1;
        border-radius: 15px; 
       padding: 6px;
       background-color: #ffffff;
        color: #000000;
        font-size: 16px;
    }
    QLineEdit:focus {
        border: 2px solid #007BFF; 
    }</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>310</y>
          <width>251</width>
          <height>35</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
        border: 1px solid #a1a1a1;
        border-radius: 15px; 
       padding: 6px;
       background-color: #ffffff;
        color: #000000;
        font-size: 16px;
    }
    QLineEdit:focus {
        border: 2px solid #007BFF; 
    }</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="LoginButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>400</y>
          <width>251</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>-1</pointsize>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #ED9455;
       border-radius: 15px; 
        color: #ffffff; 
        padding: 10px 20px; 
        font-size: 18px;
        border: none;
    }
    QPushButton:pressed {
        background-color: #0056b3; 
    }
    QPushButton:hover {
        background-color: #0056b3;
    }</string>
        </property>
        <property name="text">
         <string>Đăng nhập</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="remeberBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>350</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
        </property>
        <property name="text">
         <string>Nhớ tài khoản</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>270</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Mật khẩu</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>22</pointsize>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Sign In</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>190</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Tài khoản</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>50</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Jersey 10</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#FBC02D;</string>
        </property>
        <property name="text">
         <string>Auto Worker</string>
        </property>
       </widget>
       <widget class="QLabel" name="versionLabel">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>550</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>10</y>
         <width>511</width>
         <height>591</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFEC9E;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="gifLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>441</width>
          <height>511</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
